### ISSUE TITLE
NuGet package

### ISSUE THREAD

**Issue by zetashift** [OmanF/ArangoFoil#2]
Heyho, thanks for this wrapper, it looks nice!

In the README you state
>  (I'm sorry, but it's not likely this will ever be made into a Nuget package).
Is there any specific reason for that? Anyway we can make this possible elsewise. I'd love to help as well.

**Comment by OmanF** [comment]
Hi.

First, thank you for your interest.
Second, the reason was that when I created this code I didn't know how to pack and publish my code into Nuget packages, and I didn't think demand for such a thing would ever present itself.

But, since you asked, I've educated myself and the wrapper *is* now available as a Nuget package.

Thanks for making me step out of my comfort zone, and I wish this wrapper serves someone other than myself some good. :smiley:

P.S. I updated the `Usage` section of the README as well to include the crucial first step: defining the connection. Seems important now that I think of it.
